store/data_store.py

```diff
@@ -1,4 +1,7 @@
 import PIL
+from sampling.sample import Sample
+from multiprocessing import Queue
```

Comment:
Will this be in shared memory of all the child processes? Because that's what we want in our design.

Reply:
Yes, the entire data store is persisted.